### ISSUE TITLE
 Unknown custom element: <b-form-select-option>

### ISSUE THREAD

**Issue by vciloglu** [bootstrap-vue/bootstrap-vue#4613]
I am using Nuxt CLI. I want to create a select component. my selection tag must have a placeholder. I used the sample in the documentation for this but received the following warning : 

`[Vue warn]: Unknown custom element: <b-form-select-option> - did you register the component correctly? For recursive components, make sure to provide the "name" option.`

use a sample 

```
<template>
  <div>
    <b-form-select v-model="selected" :options="options" class="mb-3">
      <!-- This slot appears above the options from 'options' prop -->
      <template v-slot:first>
        <b-form-select-option :value="null" disabled>-- Please select an option --</b-form-select-option>
      </template>

      <!-- These options will appear after the ones from 'options' prop -->
      <b-form-select-option value="C">Option C</b-form-select-option>
      <b-form-select-option value="D">Option D</b-form-select-option>
    </b-form-select>

    <div class="mt-3">Selected: <strong>{{ selected }}</strong></div>
  </div>
</template>

<script>
  export default {
    data() {
      return {
        selected: null,
        options: [
          { value: 'A', text: 'Option A (from options prop)' },
          { value: 'B', text: 'Option B (from options prop)' }
        ]
      }
    }
  }
</script>
```

package.json 

```
"bootstrap": "^4.4.1",
"bootstrap-vue": "^2.0.0",
"nuxt": "^2.11.0",
```

**Comment by Hiws** [comment]
The `b-form-select-option` tag is new in version `2.2.0`, so you'll need to upgrade to use that specific tag.

**Comment by vciloglu** [comment]
I've tried. new version is giving me problems.

**Comment by tmorehouse** [comment]
Someone else had a similar issue and it was because they were using v2.1.0 (not v2.2.0):

https://stackoverflow.com/questions/59678515/b-form-select-option-not-working-in-bootstrap-vue

**Comment by vciloglu** [comment]
I received syntax errors when I upgraded to version 2.2.0.

I'm sure I didn't upgrade to version 2.1.0

npm install I upgraded to 2.2.0 using bootstrap-vue

**Comment by BrayanIribe** [comment]
I had the same issue. The fix was to update to ^2.2.0, I think you have another problem, apart of the bootstrap-vue.

**Comment by tmorehouse** [comment]
Upgrade to BootstrapVue `v2.2.2`

**Comment by tmorehouse** [comment]
Fixed in 2.2.0+
